notification-service/src/main/resources/database.properties

```diff
@@ -0,0 +1,3 @@
+url=jdbc:mysql://database-users.mysql.database.azure.com:3306/games?useSSL=true
```

Comment:
Can we just leave this up? not 100% sure cause repo is public now for andrew

Reply:
not much can be done with just the url if you don't have the user and pass and being exempt from the ip firewall, but ill remove it anyways since its probably better security practice

Reply:
removed now